chronosphere/intschema/generateintschema/main.go

```diff
@@ -54,6 +54,7 @@ var sharedSchemaTypeNames = map[*schema.Schema]string{
 	tfschema.TraceStringFilterSchema:      "TraceStringFilter",
 	tfschema.TraceTagFilterSchema:         "TraceTagFilter",
 	tfschema.ValueMappingsSchema:          "ValueMappings",
+	tfschema.LogPrioritiesSchema:          "LogPrioritiesSchema",
 }
```

Comment:
nit: insert new values in sorted order.

probably worth a note for future too:
```suggestion

	// Insert in sorted order.
}
```